payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/BacsDebitConfirmSpec.kt

```diff
@@ -0,0 +1,25 @@
+package com.stripe.android.ui.core.elements
+
+import androidx.annotation.RestrictTo
+import com.stripe.android.ui.core.R
+import com.stripe.android.uicore.elements.CheckboxFieldController
+import com.stripe.android.uicore.elements.CheckboxFieldElement
+import com.stripe.android.uicore.elements.IdentifierSpec
+import kotlinx.serialization.Serializable
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP_PREFIX)
+@Serializable
+class BacsDebitConfirmSpec : FormItemSpec() {
+    override val apiPath: IdentifierSpec = IdentifierSpec.Generic("")
```

Comment:
how come this doesn't have an API path?

Reply:
The field itself is required but isn't meant to be sent as part of the confirm params. This still wasn't the correct way to do this however. Updated it with a custom path and set `ignoreField` to `true` which will omit it in the params.